curriculum/challenges/english/25-front-end-development/lab-mood-board/673b3d6b7ef7318eef926d5a.md

```diff
@@ -41,111 +41,86 @@ You can use the following images in your Mood Board if you would like:
+  const img = container.querySelector(".mood-board-image");
+  assert.exists(img);
+  assert.equal(img.tagName, "IMG");
+  assert.equal(img.src, "https://cdn.freecodecamp.org/curriculum/labs/pathway.jpg");
+}
 ```
 
 Your `MoodBoardItem` component should render an `h3` element with a class of `mood-board-text` and its text set to the value of the `description` prop.
 
 ```js
-const mockedComponent = Enzyme.mount(
-  React.createElement(window.index.MoodBoardItem)
-);
-mockedComponent.setProps({ description: 'descriptionValue' });
-const h3 = mockedComponent.find('h3.mood-board-text');
-assert.lengthOf(h3, 1);
-assert.equal(h3.text(), 'descriptionValue');
+async () => {
+  const container = await __prepTestComponent(window.index.MoodBoardItem, { description: "Carribean" });
+  const text = container.querySelector(".mood-board-text");
+  assert.exists(text);
+  assert.equal(text.tagName, "H3");
+  assert.equal(text.textContent, "Carribean");
+}
 ```
 
 You should export a `MoodBoard` component.
 
 ```js
-const mockedComponent = Enzyme.mount(
-  React.createElement(window.index.MoodBoard)
-);
-assert.lengthOf(mockedComponent.find('MoodBoard'), 1);
+assert.isFunction(window.index.MoodBoard, 1);
 ```
 
 Your `MoodBoard` component should return a `div` as its top-level element.
 
 ```js
-const mockedComponent = Enzyme.mount(
-  React.createElement(window.index.MoodBoard)
-);
-assert.equal(mockedComponent.childAt(0).type(), 'div');
+const root = document.getElementById('root');
+assert.equal(root.children[0].tagName, 'DIV');
 ```
 
 Your `MoodBoard` component should render an `h1` element with a class of `mood-board-heading` and the text `Destination Mood Board`.
 
 ```js
-const mockedComponent = Enzyme.mount(
-  React.createElement(window.index.MoodBoard)
-);
-const h1 = mockedComponent.find('h1.mood-board-heading');
-assert.lengthOf(h1, 1);
-assert.equal(h1.text(), 'Destination Mood Board');
+const heading = document.getElementsByClassName('mood-board-heading')[0];
+assert.exists(heading);
+assert.equal(heading.tagName, 'H1');
+assert.equal(heading.textContent, 'Destination Mood Board');
 ```
 
-Your `MoodBoard` component should render at least three `MoodBoardItem` components, each should pass `color`, `image`, and `description` props with valid values.
+Your `MoodBoard` component should render at least three `MoodBoardItem` components.
 
 ```js
-const mockedComponent = Enzyme.mount(
-  React.createElement(window.index.MoodBoard)
-);
-const mbItems = mockedComponent.find(window.index.MoodBoardItem);
-assert.isAtLeast(mbItems.length, 3);
-const propsList = mbItems.map(item => item.props());
-propsList.forEach(({ color, image, description }) => {
-  assert.isAtLeast(color.length, 1);
-  assert.isAtLeast(image.length, 1);
-  assert.isAtLeast(description.length, 1);
-});
+const items = document.getElementsByClassName('mood-board-item');
+assert.isAtLeast(items.length, 3);                              
 ```
```

Comment:
Not sure about my changes here, any suggestions?

Reply:
Without Enzyme it's tricky to test if props are being passed to children. However, what you can do is check to see if it renders what you'd expect if those props were passed.

That means you can't directly check "did it pass the color prop?", but you can check "did the MoodBoardItem render the expected color?". i.e. you can repeat the tests you did above, but inside the loop.